bluesky_config/tiled/tld.yml

```diff
@@ -6,7 +7,6 @@ trees:
 uvicorn:
   host: 0.0.0.0
   port: 8000
-  root_path: /tiled
```

Comment:
I think we need this to expose it outside the pod?

Reply:
Yes we do. I must have removed when I was fixing up spelling mixes between `tld` and `tiled`. 

Re added and tested from inside the pod with BSUI and outside the pod via the proxy path.

Reply:
[ec3d8da](https://github.com/bluesky/bluesky-pods/pull/57/commits/ec3d8da6278eeccbed3adbdf4a536aac69c02883)